test/test_helper.rb

```diff
@@ -12,3 +12,15 @@
 require 'minitest/autorun'
 require 'minitest/unit'
 require 'minitest/pride'
+
+class Minitest::Test
```

Comment:
I would rather see the methods below added to `test/support/setup_and_teardown.rb` as a module, & then explicitly mix that module into the test classes. Monkeypatching `Minitest::Test` seems excessive.